css/_components/navbar/_navbar.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../../main.css">
+  <title>Navbar</title>
+
+  <style>
+    body {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-evenly;
+      width: 100vw;
+    }
```

Comment:
it would be better if you move internal stylesheets to external files

Reply:
Hey Mohammed, actually these are just wrappers for the components. Like these styles are not part of the component library hence I've kept them inline. The idea is that in near future I would want to bundle my CSS for performance. So if I add these redundant codes there then it will unnecessarily increase the file size and hence affect the performance. Hence, I've added them inline. I think now you're doubt might be clear.